x-pack/plugins/index_lifecycle_management/public/application/components/index_templates_flyout.tsx

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { FunctionComponent } from 'react';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { i18n } from '@kbn/i18n';
+import {
+  EuiButtonEmpty,
+  EuiFlyout,
+  EuiFlyoutBody,
+  EuiFlyoutFooter,
+  EuiFlyoutHeader,
+  EuiInMemoryTable,
+  EuiLink,
+  EuiTitle,
+} from '@elastic/eui';
+import { PolicyFromES } from '../../../common/types';
+import { useKibana } from '../../shared_imports';
+import { getTemplateDetailsLink } from '../../../../index_management/public/';
+
+interface Props {
+  policy: PolicyFromES;
+  close: () => void;
+}
+export const IndexTemplatesFlyout: FunctionComponent<Props> = ({ policy, close }) => {
+  const {
+    services: { getUrlForApp },
+  } = useKibana();
+  const getUrlForIndexTemplate = (name: string) => {
+    return getUrlForApp('management', {
+      path: `data/index_management${getTemplateDetailsLink(name)}`,
+    });
+  };
+  return (
+    <EuiFlyout onClose={close}>
+      <EuiFlyoutHeader hasBorder>
+        <EuiTitle size="m" data-test-subj="indexTemplatesFlyoutHeader">
+          <h2>
+            <FormattedMessage
+              id="xpack.indexLifecycleMgmt.policyTable.indexTemplatesFlyout.headerText"
+              defaultMessage="Index templates linked to policy {policyName}"
```

Comment:
I don't think you need to repeat _policy_ in the header. I'd be inclined to indicate what "linked" means, though.

```suggestion
              defaultMessage="Index templates that apply {policyName}"
```